misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -153,6 +154,23 @@ class JettyService @Inject internal constructor(
       server.addConnector(httpsConnector)
     }
 
+    if (webConfig.sidecar != null) {
+      val udsConnectionFactories = mutableListOf<ConnectionFactory>()
+
+      udsConnectionFactories += HttpConnectionFactory(httpConfig)
```

Comment:
should I be using the same `httpConfig` used elsewhere or should I create a uds specific one

Reply:
This doesn't answer your question directly but is related:  can or should you reuse the existing `httpConnectionFactories` defined on line 54, instead of creating a new `HttpConnectionFactory`?